docs/functions/building-functions.md

```diff
@@ -0,0 +1,42 @@
+# Building functions
+
+--8<-- "build-func-intro.md"
+
+## Local builds
+
+You can build a container image for your function locally without deploying it to a cluster, by using the `build` command.
+
+### Prerequisites
+
+- You have a Docker daemon on your local machine. This is already provided if you have used the Quickstart installation.
+
+### Procedure
+
+--8<-- "proc-building-function.md"
+
+## On-cluster Builds
```

Comment:
I wonder if this actually makes more sense in the deploy section. These are technically on-cluster builds, but the end result is a deployment, and the command used for these examples is `func deploy`.  wdyt?

Reply:
I think in terms of use cases, because it's referred to as "on cluster builds", it makes more sense that a user would look for it here. Maybe down the line the CLI command could be reworked so that it makes more sense and doesn't use `deploy`, or we could also add docs to a deploy section, but I think from a user perspective for now it makes sense that it's here.